### PR TITLE
fix(config): sm103 -> hsp02 

### LINKED ISSUE #2066: Incorrect identification of device (SM103 instead of HSP02)
**Describe the bug**

My HSP02 device (motion sensor) is incorrectly identified as SM103 (door/window contact)

**Device information**

Which device(s) is/are affected (make/model)?  
Product Manufacturer: 0x0116
Product Identifiers: 0x0001:0x0001

**Did this use to work before?**
- [x] **Yes**,  with the OpenZWave database
    
**How are you using `node-zwave-js`**
- [x] `zwavejs2mqtt` (latest) docker image

**Additional context**

this entry is the device database is wrong
https://devices.zwave-js.io/?jumpTo=0x0116:0x0001:0x0001:0.0

here are the info about the HSP02
https://products.z-wavealliance.org/products/606

it's a rebranding of the the everspring HSP02
https://devices.zwave-js.io/?jumpTo=0x0060:0x0001:0x0001:0.0

the correct configuration can also be found in OpenZWave
https://github.com/OpenZWave/open-zwave/tree/master/config/manufacturer_specific.xml#L374

### PR DESCRIPTION
**Pull request by marcus-j-davies**
fixes https://github.com/zwave-js/node-zwave-js/issues/2066

### CODE REVIEW

**Comment by marcus-j-davies** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0116/hsp02.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Re-trigger Interval Setting",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 255,
+			"defaultValue": 180,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0116/hsp02.json:83]: Does this have a unit?

File: packages/config/config/devices/0x0116/hsp02.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "LUX Level",
+			"description": "Value for what LUX level will arm the detection sensor",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0116/hsp02.json]: This should have a better label (like `Brightness Threshold`) with a description explaining that the motion sensor is only active above (or below?) that level.
`lux` should be the unit, not a name.

File: packages/config/config/devices/0x0116/hsp02.json [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "On-Off Duration",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0116/hsp02.json]: Duration sounds strongly like this should have a unit. Looking at this I don't understand what this param does, so there is probably a better label.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**